src/Features/ShippingLabelBannerDisplayRules.php

```diff
@@ -0,0 +1,192 @@
+	 * Minimum supported Jetpack version.
+	 *
+	 * @var string
+	 */
+	private $min_jetpack_version = '4.4';
+
+	/**
+	 * Supported countries by USPS, see: https://webpmt.usps.gov/pmt010.cfm
+	 *
+	 * @var array
+	 */
+	private $supported_countries = array( 'US', 'AS', 'PR', 'VI', 'GU', 'MP', 'UM', 'FM', 'MH' );
+
+	/**
+	 * Array of supported currency codes.
+	 *
+	 * @var array
+	 */
+	private $supported_currencies = array( 'USD' );
+
+	/**
+	 * Determines whether or not the banner should be displayed.
+	 *
+	 * @param string $jetpack_version Installed Jetpack version to check.
+	 */
+	public function should_display_banner( $jetpack_version ) {
+		$woocommerce_shipping_ab_active     = get_option( 'woocommerce_shipping_ab_active' );
+		$woocommerce_product_needs_shipping = false;
+		$woocommerce_store_in_us            = false;
+		$woocommerce_store_currency_in_usd  = false;
+		$base_currency                      = get_woocommerce_currency();
+		$base_location                      = wc_get_base_location();
+
+		if ( $woocommerce_shipping_ab_active ) {
+			return true;
+		}
+
+		if ( $this->banner_dismissed() ) {
+			return false;
+		}
+
+		if ( $this->jetpack_disconnected() ||
+			$this->jetpack_not_up_to_date( $jetpack_version ) ||
+			$this->wcs_not_installed() ||
+			$this->ups_not_installed() ||
+			$this->fedex_not_installed() ||
+			$this->shipstation_not_installed() ||
+			$this->shippingeasy_not_installed() ) {
+			return false;
+		}
+
+		$order = wc_get_order( get_post()->ID );
+
+		if ( ! $order ) {
+			return false;
+		}
+		// At this point (no packaging data), only show if there's at least one existing and shippable product.
+		foreach ( $order->get_items() as $item ) {
+			if ( $item instanceof \WC_Order_Item_Product ) {
+				$product = $item->get_product();
+
+				if ( $product && $product->needs_shipping() ) {
+					$woocommerce_product_needs_shipping = true;
+				}
+			}
+		}
+
+		$woocommerce_store_currency_in_usd = $this->is_supported_currency( $base_currency );
+
+		$woocommerce_store_in_us = $this->is_supported_country( $base_location['country'] );
+
+		return $woocommerce_product_needs_shipping && $woocommerce_store_currency_in_usd && $woocommerce_store_in_us;
+	}
+
+	/**
+	 * Check if the banner was dismissed by the user
+	 *
+	 * @return bool
+	 */
+	private function banner_dismissed() {
+		$dismissed_timestamp = get_option( 'woocommerce_shipping_dismissed_timestamp' );
+
+		$dismissed_for_good = -1 === $dismissed_timestamp;
+		$dismissed_24h      = time() < $dismissed_timestamp;
+
+		return $dismissed_for_good || $dismissed_24h;
+	}
+
+	/**
+	 * Check if WooCommerce Services is inactive
+	 *
+	 * @return bool
+	 */
+	private function wcs_not_installed() {
+		return ! is_plugin_active( 'woocommerce-services/woocommerce-services.php' );
+	}
+
+	/**
+	 * Check if jetpack is inactive or disconnected
+	 *
+	 * @return bool
+	 */
+	private function jetpack_disconnected() {
+		return ! is_plugin_active( 'jetpack/jetpack.php' );
```

Comment:
This is only checking if the plugin is installed which does not match the method name or the comments. 
Looks like we only need to check if it is installed according to https://github.com/Automattic/woocommerce-shipping-issues/issues/13#issuecomment-594845206
So I'd just change the method name to `jetpack_installed`